config/constants.js

```diff
@@ -0,0 +1,3 @@
+const DB_URL = `mongodb+srv://user:user@cluster0.09sqx.mongodb.net/myFirstDatabase?retryWrites=true&w=majority`;
+
+export default DB_URL;
```

Comment:
It's a common file for all constants, we can export each variable separately.
```
export const DB_URL = `...`;
export const TEST_EXAMPLE = '...'
```
And import them in future:
`import { DB_URL, TEST_EXAMPLE } from 'config/constants';`